xmipp

```diff
@@ -449,7 +449,9 @@ def compile_libsvm():
     ok = runJob("make lib")
     libDir = "src/xmipp/lib"
     os.chdir(currDir)
-    return ok and runJob("cp " + libsvmDir + "/libsvm.so.2" + " " + libDir+"/libsvm.so")
+    ok = ok and runJob("cp " + libsvmDir + "/libsvm.so.2" + " " + libDir)
```

Comment:
If src/xmipp/lib/libsvm.so exist, fail to create ln. Before try it remove the file.
`    ok = ok and runJob("rm " + libDir + "/libsvm.so")`
If is a file, it remove the old file, and if it is a ln, will be create again the next line.

Reply:
Let us see now. @albertmena, Alberto, do you continue with it?

Reply:
For me, the R[emoval of link problem ](https://github.com/I2PC/xmipp/pull/471/commits/6a248c3c7110202d7c0224cc04316ffd439ca4fc)fix the PR.